android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/NearbyBeaconsFragment.java

```diff
@@ -413,7 +413,8 @@ public void addItem(PwPair pwPair) {
       // If isResolvableDevice place in the folder at the bottom
       // of the list (making the folder if it didn't already exist)
       // Otherwise place in the bottom of the non-folder list
-      if (Utils.isResolvableDevice(pwPair.getUrlDevice())) {
+      if (Utils.isResolvableDevice(pwPair.getUrlDevice())
+          || Utils.isWifiDirect(pwPair.getUrlDevice())) {
```

Comment:
Could you add a comment explaining why this is necessary?  i don't see why it would be